src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/AlertCodesController.kt

```diff
@@ -69,6 +71,42 @@ class AlertCodesController(
     httpRequest: HttpServletRequest,
   ): AlertCode = alertCodeService.createAlertCode(createAlertCodeRequest, httpRequest.alertRequestContext())
 
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @PreAuthorize("hasAnyRole('$ROLE_ALERTS_ADMIN')")
+  @DeleteMapping("/{alertCode}")
+  @Operation(
+    summary = "Create an alert code",
+    description = "Create a new alert code, typically from the Alerts UI",
```

Comment:
Should probably be changed to deactivate rather than create

Reply:
Done.